R/R/class.R

```diff
@@ -7,7 +7,7 @@
 #' @return An object of class \code{feather}
 #' @export
 feather <- function(path) {
-  path <- normalizePath(path, mustWork = TRUE)
+  path <- enc2native(path)
```

Comment:
This is not an adequate replacement to `normalisePath()` as it won't replace `~`

Reply:
Oh okay - new commit fixes the error but still normalizes. (FWIW the old code followed readxl's lead - https://github.com/tidyverse/readxl/pull/477 takes out `normalizePath()`)